SpatialGDK/Documentation/content/get-started/example-project/exampleproject-play.md

```diff
@@ -1,48 +1,63 @@
+The list will look something like this: </br>
+![img]({{assetRoot}}assets/tutorial/console-deployments-list.png)<br/>
+_Image: The Console project screen deployment list_
+</br></br>
+1. Select your deployment name to see its overview screen.</br>
+The overview screen has a **Start** button  near the top. Select it to bring up the Start deployment dialogue box.
+</br></br>
+1. In the Start deployment dialogue box, select **Start**. 
+</br>
+Now the deployment is running, you see this near the top of the deloyment overview screen:
+![img]({{assetRoot}}assets/tutorial/console-launcher-button.png)<br/>
+_Image: The Console deployment overview screen showing the Launcher button_
+</br></br>
+1. In the deployment overview screeb, select the blue **Launch** button to open the Launcher dialog box.</br>
+It looks like this:</br>
 ![img]({{assetRoot}}assets/tutorial/launch.png)<br/>
-_Image: The SpatialOS console launch window_
-
-> **TIP:** Check out the [cloud deployment workflow page]({{urlRoot}}/content/cloud-deployment-workflow) for a reference diagram of this workflow.
-
-Once the client has launched, enter a name for your player character and select **Start** to start playing. <br/>
-
+_Image: The Console's Launcher dialog box_
+</br></br>
+1. Select **Launch** in the Launcher dialog box to download a game client.</br></br>
+1. Once the client has launched, enter a name for your player character and select **Start** to start playing. <br/></br>
 ![img]({{assetRoot}}assets/example-project/example-project-lobby.png)<br/>
 _Image: The Example project lobby screen_
 
 ### Step 2: Share your game
 To share your cloud deployment: 
 
-1. Open the Console and select your deployment by the name you gave it when you ran the `spatial cloud launch` command. 
-1. Select **Share** on the right-hand side of the screen. 
-1. Accept the terms and conditions.
-
-![img]({{assetRoot}}assets/example-project/example-project-share-tos.png))<br/>
-
+1. In the Console, go to the deployment overview screen.</br>
+The screen has this near the top.
+![img]({{assetRoot}}assets/tutorial/console-launcher-button.png)<br/>
+_Image: The Console deployment overview screen showing the white **Share** button_
+<br/><br/>
+1. Select **Share** to see the Share Application screen.<br/><br/>
+1. In the Share Application screen, check the box to accept the terms and conditions and select **Get Share URL**..
+![img]({{assetRoot}}assets/example-project/example-project-share-tos.png)<br/>
 _Image: The SpatialOS Console Share Application terms of service screen_
-
-After you have accepted the terms and conditions, you can send your share URL to other people so they can try out your game. 
-
-![img]({{assetRoot}}assets/example-project/example-project-share-screen.png)<br/>
-
+<br/><br/>
+1. You can now see A Token URL. You can share this to game testers so they can try out your game. 
```

Comment:
Is "A Token URL" correct?

Reply:
That doesn't make sense. It's a URL which is a token but it doesn't really matter.